packages/race-cli/src/reporters/aggregate-console-reporter.ts

```diff
@@ -1,44 +1,57 @@
 import {std, mean, round} from 'mathjs';
 import fs from 'fs/promises';
 import json2md from 'json2md';
-import {LighthouseResultsWrapper} from '@racepoint/shared';
+import {UserFlowStep, UserFlowResultsWrapper} from '@racepoint/shared';
 import {
   BaseRacepointReporter,
   LightHouseAuditKeys,
   ProfileConfig,
+  MathOperation,
+  StepData,
+  StepDataCollection,
+  ComputedStepDataCollection,
+  LabeledStepDataCollection,
 } from '../types';
 import {MissCount, CacheMetricData} from '@racepoint/shared';
 import logger from '../logger';
 import {formatFilename} from '../helpers';
 
 const STD_DEVIATION_KEY = 'Standard Deviation';
 const MEAN_KEY = 'Mean';
-const METRIC_KEY = 'Metric';
+const METRIC_KEY = 'Measurement';
 const FORMAT_MD = 'md';
 
 const resultsToMarkdown = (
```

Comment:
beyond the scope of this PR but something to keep in mind that we might want to break out the markdown, html, json, console, etc "formats" out into a separate 'output formats' file. just a thought